inspirehep/modules/theme/jinja2filters.py

```diff
@@ -488,6 +488,9 @@ def publication_info(record):
     pub_infos = []
     if 'publication_info' in record:
         for pub_info in record.publication_information:
+            if pub_info.get('hidden'):
+                continue
+
```

Comment:
Actually, I'm considering removing it altogether: this is not the right way to interact with our schema. We need to have a smart getter that does this job for us that encapsulates the notion of "don't show hidden `publication_info`s" for us.